scripts/check.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+set -ex
+
+poetry run ruff format --check bayes_opt tests
+poetry run ruff check bayes_opt tests
```

Comment:
What exactly is the purpose of the scripts? How do you envision the workflow to be?

Reply:
Although `pre-commit` is useful,
there are many times when I want to use commands directly before committing.
This is a simple script for those times.

If it works as intended,
This script should guarantee the same result as `pre-commit`.

Reply:
if it's just that, you can do `pre-commit run` to run it manually

Reply:
> if it's just that, you can do `pre-commit run` to run it manually

That's right, that's what I usually do.
`poetry run pre-commit run --all-files --show-diff-on-failure`

But sometimes, when there are non-commit related modifications, 
`pre-commit` shows logs that are not needed.
And it seemed tricky to control.

Reply:
Is there a reason why you did not similarly add a script to excute tests & calculate coverage, e.g. `poetry run pytest --cov-report xml --cov=bayes_opt/`?

Reply:
> Is there a reason why you did not similarly add a script to excute tests & calculate coverage, e.g. `poetry run pytest --cov-report xml --cov=bayes_opt/`?

I didn't add the script because the test took a long time.(>60s)

I wanted the scripts to be small enough that anyone could just run them whenever they wanted and see what happened.